### ISSUE TITLE
Frame rate pausing when in game panel enabled

### ISSUE THREAD

**Issue by JDS310** [nguyenquyhy/Flight-Events#12]
Love the concept, but FS2020 is unplayable when the in game panel is enabled due to constant frame rate pausing/hanging every 1-2 seconds. I've only installed and tested since the latest patch. Issue occurs regardless of whether the panel is enabled or not within FS2020. I'm running a 2080Ti at 2560x1440 and changing graphics settings from down from ultra mode reduces the impact but doesn't stop it. Removing the panel from the community folder is the only way to stop the problem.

My alternate strategy was just to have a browser window open separately to FS2020 and just set the browser to follow my plane, but the follow is "forgotten" every 30 mins or so and it appears only the variables that can be sent in msfs mode to https://events.flighttracker.tech/ is latitude and longitude and not the plane details.

Would love this to work as it is an awesome feature, but need either the in-game panel not to create the stuttering or for the website to allow follow mode hold for the browser session.

**Comment by nguyenquyhy** [comment]
Yes I will work on improving the panel this weekend. The current version is more or less an exact embed of the website, so that wouldn't work very well in an environment with limited processing resource in-game.

The main problem right now would be the very high update of > 100 aircraft. That works ok in browser with dedicated process for each tab but turns out to be quite heavy in-game. I am thinking of adding some kinds of filter so that not all aircraft show up, or throttling the update rate.

I have also heard about the browser forgetting the Follow setting, I'll also check that out this weekend.

And thanks for bearing with the very rough first version of the panel and giving me the feedback.

**Comment by JDS310** [comment]
Hey Nguyễn, happy to test. I use the Simionic MFD and PPD panels (on an iPad), but FlightTracker is such a better interface than the MFD panel so happy to test and provide feedback as needed.
